ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/ProjectNotes/NewNote.cshtml.cs

```diff
@@ -1,32 +1,20 @@
-using ApplyToBecome.Data;
-using ApplyToBecome.Data.Models.ProjectNotes;
-using ApplyToBecome.Data.Services;
-using ApplyToBecomeInternal.Models;
-using Microsoft.AspNetCore.Mvc;
-
-namespace ApplyToBecomeInternal.Pages.ProjectNotes
-{
-	public class NewNoteModel : BaseAcademyConversionProjectPageModel
-	{
-		private readonly IProjectNotes _projectNotes;
-
-		public NewNoteModel(IAcademyConversionProjectRepository repository, IProjectNotes projectNotes) : base(repository)
-		{
-			_projectNotes = projectNotes;
-		}
-
-		[BindProperty]
-		public string subject { get; set; }
-
-		[BindProperty]
-		public string body { get; set; }
-
-		public IActionResult OnPost(int id)
-		{
-			var note = new ProjectNote(subject, body);
-			_projectNotes.SaveNote(id, note);
-			TempData["newNote"] = true;
-			return RedirectToPage(Links.ProjectNotes.Index.Page, new { id = id });
-		}
-	}
-}
+using ApplyToBecome.Data.Services;
+using ApplyToBecomeInternal.Services;
+using Microsoft.AspNetCore.Mvc;
+using System.Threading.Tasks;
+
+namespace ApplyToBecomeInternal.Pages.ProjectNotes
+{
+	public class NewNoteModel : UpdateAcademyConversionProjectPageModel
+	{
+		public NewNoteModel(IAcademyConversionProjectRepository repository, ErrorService errorService) : base(repository, errorService)
+		{
+		}
+
+		public override async Task<IActionResult> OnPostAsync(int id)
+		{
+			TempData["newNote"] = true;
+			return await base.OnPostAsync(id);
+		}
+	}
+}

```

Comment:
I would either carry on using the properties that were here before or separate the project note properties from the ACPPostModel as ACP and project note are separate entities